config/rabbitmq/rabbitmq-http.go

```diff
@@ -10,12 +10,11 @@ import (
 	"io/ioutil"
```

Comment:
@pct960 are we using this go file? I thought we were using the one in your repo?

Reply:
We are using the one from my repo. The script makes a local copy of it.